providers/google/tests/system/google/cloud/cloud_build/example_cloud_build.py

```diff
@@ -59,17 +59,22 @@
 # [START howto_operator_gcp_create_build_from_storage_body]
 CREATE_BUILD_FROM_STORAGE_BODY = {
     "source": {"storage_source": GCP_SOURCE_ARCHIVE_URL},
-    "steps": [{"name": "ubuntu", "args": ["echo", "Hello world", "sleep 200"]}],
+    "steps": [{"name": "ubuntu", "entrypoint": "bash", "args": ["-c", "echo Hello world && sleep 200"]}],
 }
 # [END howto_operator_gcp_create_build_from_storage_body]
 
 # [START howto_operator_create_build_from_repo_body]
 CREATE_BUILD_FROM_REPO_BODY: dict[str, Any] = {
     "source": {"repo_source": {"repo_name": GCP_SOURCE_REPOSITORY_NAME, "branch_name": "master"}},
-    "steps": [{"name": "ubuntu", "args": ["echo", "Hello world", "sleep 200"]}],
+    "steps": [{"name": "ubuntu", "entrypoint": "bash", "args": ["-c", "echo Hello world && sleep 200"]}],
 }
 # [END howto_operator_create_build_from_repo_body]
 
+CREATE_BUILD_FROM_REPO_NO_WAIT_BODY: dict[str, Any] = {
```

Comment:
the variable name says `NO_WAIT` but we have `sleep 300` in the steps.
Is this related? or does `NO_WAIT` mean a totally different thing?